adserver/api/views.py

```diff
@@ -157,7 +157,15 @@ class AdDecisionView(GeoIpMixin, APIView):
     renderer_classes = (JSONRenderer, JSONPRenderer)
 
     def _prepare_response(
-        self, ad, placement, publisher, keywords, url, forced=False, paid_eligible=False
+        self,
+        ad,
+        placement,
+        publisher,
+        keywords,
+        url,
+        forced=False,
+        paid_eligible=False,
+        rotated=False,
```

Comment:
Thoughts on using `kwargs` for the default args?

Reply:
Seems fine.. just to make the function sig smaller?